workspaces/mama/src/ManifestManager.class.ts

```diff
@@ -201,7 +209,7 @@ export class ManifestManager<
     }
 
     const packageLocation = location.endsWith("package.json") ?
-      location :
+      path.join(path.dirname(location), "package.json") :
```

Comment:
No we should not touch here, there is no relation with this.relation alloced in the constructor (that where the assert should be done)